be/src/vec/olap/olap_data_convertor.cpp

```diff
@@ -587,28 +594,17 @@ const void* OlapBlockDataConvertor::OlapColumnDataConvertorVarChar::get_data_at(
     return null_flag ? nullptr : _slice.data() + offset;
 }
 
-Status OlapBlockDataConvertor::OlapColumnDataConvertorVarChar::convert_to_olap() {
-    assert(_typed_column.column);
-    const vectorized::ColumnString* column_string = nullptr;
-    if (_nullmap) {
-        auto nullable_column =
-                assert_cast<const vectorized::ColumnNullable*>(_typed_column.column.get());
-        column_string = assert_cast<const vectorized::ColumnString*>(
-                nullable_column->get_nested_column_ptr().get());
-    } else {
-        column_string = assert_cast<const vectorized::ColumnString*>(_typed_column.column.get());
-    }
-
+Status OlapBlockDataConvertor::OlapColumnDataConvertorVarChar::convert_to_olap(
```

Comment:
Why change OlapColumnDataConvertorVarChar?

Reply:
extract the common component used for `OlapColumnDataConvertorVariant`